frontend/src/Editor/LeftSidebar/SidebarGlobalSettings.jsx

```diff
@@ -17,6 +25,13 @@ export const LeftSidebarGlobalSettings = ({ globalSettings, globalSettingsChange
   };
   return (
     <>
+      <Confirm
+        show={showConfirmation}
+        message={`Do you want to ${is_maintenance_on ? 'disable' : 'enable'} maintenance ?`}
```

Comment:
Please change the message to `Users will not be able to launch the app until maintenance mode is turned off, do you wish to continue?`

When it is turned off, `Users will now be able to launch the released version of this app, do you wish to continue`?